src/helpers/generateId.ts

```diff
@@ -0,0 +1,7 @@
+export const generateUniqueId = () => {
+  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function (c) {
```

Comment:
In case you don't need to support some older browser (like here) I recommend `window.crypto.randomUUID()` instead :D